plugin/src/main/java/io/jenkins/plugins/casc/model/Mapping.java

```diff
@@ -74,4 +73,8 @@ public Mapping clone() {
         });
         return clone;
     }
+
+    public static Mapping getEmpty() {
```

Comment:
I would just call it empty, you aren’t getting a field on a specific instance

Reply:
Good idea.